bitbots_docs/docs/manual/tutorials/basler_camera_setting.rst

```diff
@@ -0,0 +1,88 @@
+You need to balance the exposure and gain a bit to trade-off blur and noise while remaining the correct brightness.
+
+If the image is too bright, turn down gain and exposure. Here you also need to trade off the different errors.
+
+Sadly the parameters can not be dynamically reconfigured. So you need to restart the driver every time you change something.
+To speed this a bit up, you can start the driver without the vision by typing
+
+``roslaunch bitbots_bringup basler_camera.launch``.
+
+Make sure no other driver instances are running in the background.
+
+
+Camera Calibration
+------------------
+
+The camera calibration is loaded automatically depending on the robot's name. If you connect a camera locally a default calibration is loaded.
+
+The calibration files are name following the following pattern ``basler_drivers/pylon_camera/config/camera_calibration_<name>.yaml``.
+
+To create new calibration files see
+
+.. _`ROS camera calibration`: http://wiki.ros.org/camera_calibration/Tutorials/MonocularCalibration
+
+White balance
+-------------
+
+In case of a correct brightness, but wrong looking colors have a look at the following part.
+
+This part should be tweaked after all brightness and gain settings because they could possibly affect the color temperature too.
+
+Our vision pipeline includes a white balancing node on the ROS level.
+Cameras need to adjust their white balance to correct the effects of different light temperatures.
+E.g. a light bulb casts a different light color as the sun.
+Most cameras do this automatically, but wrong values could result in unusable images for the vision.
+Therefore we set this value manually.
+
+The light temperature can be set in the dynamic reconfigure UI of the *whiteblancer*.
+This should be used to find the best settings by tweaking the parameter a bit until the colors nearly look like the colors in real life.
+
+A good orientation should bee white objects in the image, which should be displayed white.
+
+Note: Sometimes human vision recognizes slightly blue-white tones whiter than the real white.
+So if the object appears unnaturally white, turn the setting a bit more in the warmer direction even if the white looks a bit grayish now. This is a brightness issue.
+
+
+After tweaking the settings the estimated value can be written in the following config file under the name *temp*.
+
+``bitbots_vision/white_balancer/config/config.yaml``
+
+Some good indoor value should be something like 3000 K and for outdoor applications something about 6500 K.
+
+
+Important Notes!
+----------------
+
+All robots have individual cameras!
+Do not switch the camera without changing the name of the calibration file and the name of the camera itself in the Pylon GUI.
+Otherwise, the robot will not connect to the camera, which is resulting in a blind robot.
+
+This is implemented to fix cross over issues related to two robots being in the same network and both using their cameras over ethernet.
```

Comment:
This part seems to me like it should be near the top of the document.
If the camera on my robot does not work, I should be able to find it without looking at the previous parts, otherwise i can just skip over this part.

Reply:
I thought that too, but this is a camera settings tutorial and this part is a little bit off topic.
I moved this to a special waring box at the top.